build.sbt

```diff
@@ -1,13 +1,16 @@
 import com.typesafe.sbt.SbtNativePackager.autoImport.maintainer
 import com.typesafe.sbt.packager.archetypes.ServerLoader.Systemd
 import com.typesafe.sbt.packager.debian.DebianPlugin.autoImport.debianPackageDependencies
-import sbt.Keys.organization
+import sbt.Keys._
 
 lazy val commonSettings = Seq(
   scalaVersion in ThisBuild := "2.11.8",
   organization in ThisBuild := "com.gu",
 
-  resolvers += "Guardian Bintray" at "https://dl.bintray.com/guardian/editorial-tools",
+  resolvers ++= Seq("Guardian Bintray" at "https://dl.bintray.com/guardian/editorial-tools",
+    "Sonatype OSS Snapshots" at "http://oss.sonatype.org/content/repositories/snapshots/",
+    Resolver.sonatypeRepo("snapshots")
+  ),
```

Comment:
Perhaps we should check the `Sonatype releases` before snapshots. Quite a few Guardian repos do:
```
"Sonatype OSS" at "http://oss.sonatype.org/content/repositories/releases/",
"Sonatype OSS Snapshots" at "http://oss.sonatype.org/content/repositories/snapshots/",
```